src/Summoner/Template/Stack.hs

```diff
@@ -9,11 +9,13 @@ import NeatInterpolation (text)
 import Summoner.GhcVer (GhcVer (..), baseVer, latestLts, showGhcVer)
 import Summoner.Settings (Settings (..))
 import Summoner.Tree (TreeFs (..))
+import Summoner.Default (defaultGHC)
 
 
 stackFiles :: Settings -> [TreeFs]
-stackFiles Settings{..} = map createStackYaml settingsTestedVersions
+stackFiles Settings{..} = map createStackYaml ghcVersions
```

Comment:
`memptyIfFalse settingsStack` still should be used here because `ghcVersions` is not an empty list and we don't wont to generate `stack.yaml` files if user didn't ask for them 🙂